### PR TITLE
Jc editmessage

### PR DESCRIPTION
**Pull request by jeffreychen2016**

### CODE REVIEW

File: javascripts/edit.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const getCurrentTime = require('./timestamp');
+
+const retrieveMessage = (e) => {
+  if (e.target.classList[2] === 'btn-message-edit') {
```
Comment (johnachor) [inline javascripts/edit.js]: you can do 
```[...e.target.classList].includes('btn-message-edit')```

here, and it won't be dependent on the order of the class list

**Comment by amillion3** [comment]
Two ideas:
- When you hit the 'Edit' button, it would be great if the input box received focus.
    - https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus
- When you hit the 'Edit' button, it would be great if that specific message was highlighted or somehow identified as the message being edited. Maybe just add a thick border or add some kind of drop shadow? 

These might be pushing into stretch goals, but they seem like worthwhile features.

Looks great though!

File: javascripts/edit.js (hunk 1 above)
Comment (jeffreychen2016) [reply javascripts/edit.js]: cool, good to know. will change my codes. Thx 😃

**Comment by amillion3** [comment]
👍

**Comment by ASlayton** [comment]
👍

**Pull request merged**